utils/network.go

```diff
@@ -59,6 +59,16 @@ func DoNetworking(args *skel.CmdArgs, conf types.NetConf, result *current.Result
 			return err
 		}
 
+		if mac, err := net.ParseMAC("EE:EE:EE:EE:EE:EE"); err != nil {
+			logger.Infof("failed to parse MAC Address: %v. Using kernel generated MAC.", err)
+		} else {
+			// Set the MAC address on the host side interface so the kernel does not
+			// have to generate a persistent address which fails some times.
+			if err = netlink.LinkSetHardwareAddr(hostVeth, mac); err != nil {
+				logger.Infof("failed to Set MAC of %q: %v. Using kernel generated MAC.", hostVethName, err)
```

Comment:
I think this one should probably be a Warning log

Reply:
nit: should probably be `set` rather than `Set`